src/OpenTelemetry.AutoInstrumentation/Configuration/ConfigurationKeys.cs

```diff
@@ -92,6 +92,11 @@ public class ConfigurationKeys
     /// </summary>
     public const string LogDirectory = "OTEL_DOTNET_AUTO_LOG_DIRECTORY";
 
+    /// <summary>
+    /// Configuration key for enabling the notification of unhandled exceptions.
+    /// </summary>
+    public const string UnhandledExceptionsEnabled = "OTEL_DOTNET_AUTO_UNHANDLEDEXCEPTION_ENABLED";
```

Comment:
No constructive proposal at this moment but we should try to came up with a better name for the env var.

Reply:
Agreed, I just wanted to get something out there start the conversation.
Some ideas:
- NOTICE_UNHANDLEDEXCEPTIONS
- PROCESS_UNHANDLEDEXCEPTIONS
- TROUBLESHOOT_UNHANDLEDEXCEPTIONS

Some things that I think we should avoid:
- Names that imply that we are capturing and reporting the unhandled exception because I don't think that we can guaranty that it will be added to any piece of telemetry other than our own logs.
- Names that are overloaded and used to represent other OpenTelemetry concepts.

Reply:
Another suggestion: FLUSH_ON_UNHANDLEDEXCEPTION

Reply:
I thought about something specific like `FLUSH_ON_UNHANDLEDEXCEPTION`. I did not go with something like that initially because I wanted to leave open the possibility of doing other things like actually attempting to associate that exception to a span. However, it might make more sense to control that future possibility with a separate environment variable. Having a separate setting for that type of behavior may also be good to allow people to control how much risk they want to take on with these types of settings.